### ISSUE TITLE
Can I train slot filter or introduce a RegEx?

### ISSUE THREAD

**Issue by paulagd** [deepmipt/DeepPavlov#1193]
Hi, I am trying to use the slot filter to make it recognise **megapixels** slot but in my database I do not have all kind of megapixels. Can I make it learn that the previous number from "megapixels" word or "Megapixels" word are actually being megapixels or is it the only way to put all kind of options in the **slot_values config** file?

PS: I am trying to modify [go_bot_extended tutorial ](https://github.com/deepmipt/DeepPavlov/blob/master/examples/gobot_extended_tutorial.ipynb)to my own domine.

Thank you a lot!

**Comment by oserikov** [comment]
Hey! It seems like one could replace the slot value with some mock value before applying the bot. And then to restore the original value in the response when needed.

**Comment by paulagd** [comment]
Hey! 

Thank you for the response!!  However, I am understanding the second part -  restoring the original result -  but the part that I do not not how to build is the first one: how do I set a mock value? For example, if I set 2 as mock megapixels value, it won't be able to detect other megapixels no?

I would like to know how I can detect any number before "megapixels" and then of course, I can restore the original response. 

Thank you very much!
